### PR TITLE
add(forms) Added the 'accept' attribute for file inputs

### PR DESCRIPTION
**Pull request by sander-c**
## PR Checklist

This PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/antwerp-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/antwerp-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by SanneVerbist — APPROVED** [review]

File: packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.html [hunk 1]
```diff
@@ -32,6 +36,11 @@
...
         type="button">
   <span [id]="ariaId"><ng-content select=".m-upload__button"></ng-content></span>
-  <input #fileInput (change)="updateFiles()" [attr.aria-labelledby]="ariaId" [disabled]="disabled" [id]="id"
-         class="m-upload__input" multiple="multiple" type="file">
+  <input
+    #fileInput
+    type="file"
+    class="m-upload__input"
+    [id]="id"
+    [attr.aria-labelledby]="ariaId"
```
Comment (TriangleJuice) [inline packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.html:44]: Same here.

File: packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.ts [hunk 2]
```diff
@@ -7,14 +7,15 @@ import {Uploader} from '../../classes/uploader.class';
...
-export class UploadZoneComponent {
+export class UploadZoneComponent implements AfterViewInit {
   @ViewChild('fileInput', {static: false}) fileInput: ElementRef;
 
   @Input() public uploader: Uploader;
   @Input() public disabled = false;
   @Input() public multiple = true;
   @Input() public id = '';
   @Input() public ariaId = '';
+  @Input() public accept = [];
```
Comment (TriangleJuice) [inline packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.ts:18]: I'm missing the following two lines here:
```
@Input() public disabled = false;
@Input() public multiple = true;
```

File: packages/styleguide/src/examples/pages/aui-forms/upload/upload.page.ts [hunk 3]
```diff
@@ -40,8 +41,11 @@ export class FormsUploadDemoPageComponent {
 
 export class AppModule {};`;
 
-  public uploadExampleJS1 = `public dropzone1: UploadOptions = {
-	allowedMimeTypes: ['image/jpeg'],
+//   public uploadExampleJS1 = `public dropzone1: UploadOptions = {
+// 	allowedMimeTypes: ['image/jpeg'],
```
Comment (TriangleJuice) [inline packages/styleguide/src/examples/pages/aui-forms/upload/upload.page.ts:45]: Please remove ;-)

**Review by TriangleJuice — CHANGES_REQUESTED** [review]
Thanks for the work! I only have some small remarks :-)

File: packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.ts (hunk 2 above)
Comment (TriangleJuice) [reply packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.ts:18]: Excuses, de inputs waren reeds aanwezig.

**Review by TriangleJuice — APPROVED** [review]

**Pull request merged**